hack/run-tests.sh

```diff
@@ -0,0 +1,464 @@
+#!/bin/bash
```

Comment:
Can we maybe share this file as we do with feature files? This way we would have to make every change twice. Same goes for the tests of this script.

Reply:
It would be nice to share those files, though I have no idea how to it.
@radtriste Any idea?

Reply:
we could have one here which acts as proxy and call the `kogito-operator` one with same arguments I guess.
For the makefile, we could have our own Makefile in `kogito-operator` test folder and again retrieve and call it from this Makefile here.
wdyt ? 
I could implement this tomorrow if needed :)

Reply:
Sounds good to me, feel free to prepare it if you have time :)